nimbus/src/evaluator.rs

```diff
@@ -362,5 +373,20 @@ mod tests {
         assert!(
             matches!(enrollment.status, EnrollmentStatus::Enrolled { reason: EnrolledReason::Qualified, .. })
         );
+
+        // Fits because of the normandy_id.
+        let id = uuid::Uuid::parse_str("9d275791-3f31-4549-b30c-e32c750e4787").unwrap();
+        let available_randomization_units =
+            AvailableRandomizationUnits::with_normandy_id(&id.to_string());
+        let enrollment = evaluate_enrollment(
+            &id,
+            &available_randomization_units,
+            &Default::default(),
+            &experiment3,
+        )
+        .unwrap();
+        assert!(
+            matches!(enrollment.status, EnrollmentStatus::Enrolled { reason: EnrolledReason::Qualified, .. })
+        );
```

Comment:
Please consider also adding a test for the error case, that if the normandy_id is not available then we end up with `EnrollmentStatus::Error`.